app/models/tweet.rb

```diff
@@ -1,3 +1,5 @@
 class Tweet < ApplicationRecord
-	validates :tweet, presence: true
+  belongs_to :user
+  validates :user_id, presence: true
```

Comment:
No need for this. `has_many` associations in rails are `required: true` by default.
In any case, validating foreign keys in the model is weird.